docs/snippets/change-a-currency-symbol.md

```diff
@@ -0,0 +1,28 @@
+# Rename a country
+
+See the [currency list](https://woocommerce.github.io/code-reference/files/woocommerce-includes-wc-core-functions.html#source-view.475) for reference on currency codes.
+
+
+> This is a **Developer level** doc. If you are unfamiliar with code and resolving potential conflicts, select a [WooExpert or Developer](https://woocommerce.com/customizations/) for assistance. We are unable to provide support for customizations under our [Support Policy](http://www.woocommerce.com/support-policy/).
```

Comment:
Same as noted elsewhere, we can drop this warning:

```suggestion
```

Reply:
Removing this note based on p1692723749470709/1692723559.960259-slack-C0516GKR69W